src/modules/ourPopUp.js

```diff
@@ -0,0 +1,40 @@
+import {
+  formComment, showComment, getAddedComments,
+} from './ourComments.js';
+
+const popupContainer = document.querySelector('.popup-display');
+
+const renderPopup = async (
+  id,
+  image,
+  name,
+  language,
+  genres,
+  rating,
+  schedule,
+) => {
+  popupContainer.innerHTML = '';
+
+  popupContainer.innerHTML += `
+   <div class="show-${id} movie-img-sect">
+     <div class="close-icon">X</div>
+     <img src=${image.original} alt="" class="movie-img" />
+     <h2 class="movie-pop-title display-comments">${name}</h2>
+     </div>
+   <div class="movie-features">
+   <h3 class="padd movie-info">Movie Info</h3>
+     <p class="padd" >Language: ${language}</p>
+     <p class="padd" >Genre: ${genres.join(',')}</p> 
+     <p class="padd">Rating: ${rating.average} <i class="fa-solid fa-star"></i></p>
+     <p class="padd" >Schedule: ${schedule.time} on ${schedule.days}s</p>
+     <div class="comments-container"></div
+ `;
```

Comment:
- [ ]  Hello, can you follow the original wireframe to design the pop-up page? Because the requirements say you should follow the layout of the wireframes provided.
- Kindly send the movie's info and comments section below the picture. 
- Your project:
![Capture](https://github.com/tatouroberto/javascriptCapstone/assets/63972670/4585a510-41e1-45a6-9f49-0a1b5d376262)
- Original




![1](https://github.com/tatouroberto/javascriptCapstone/assets/63972670/6f0044a3-8a0d-490b-9fd9-798931388954)

Reply:
fixed